recommender/recommender.py

```diff
@@ -41,13 +66,29 @@ class RecommenderXBlock(XBlock):
     by students; they can also vote for useful resources and flag problematic
     ones.
     """
-    default_recommendations = List(
-        help="List of default help resources", default=[], scope=Scope.content
+    seen = Boolean(
+        help="Has the interacted with the XBlock before?",
+        default=False,
+        scope=Scope.user_info
     )
-    # A list of default recommenations, it is a JSON object across all users,
+
+    recommender_version = String(
+        help="The version of this RecommenderXBlock",
+        default="recommender.v1.0",
+        scope=Scope.content
+    )
+
+    intro_enabled = Boolean(
+        help="Take users on a little tour the first time they see the XBlock?", default=True, scope=Scope.content
```

Comment:
We cleared up the text. It is, actually, only the first time they see it. The problem we're trying to solve is discoverability. We explicitly point out the Recommender the first time the user runs across it, and give a short tour of what it does.